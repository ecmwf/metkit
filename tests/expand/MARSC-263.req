retrieve,
    levelist=0/5,
    section=h,
    stream=ocea,
    system=3,
    method=1,
    number=0/to/4,
    time=00,
    levtype=dp,
    product=inst,
    expver=1,
    param=129/130/145,
    class=od,
    date=20120101,
    type=of,
    step=24/to/96/by/24,
    expect=60,
    database=marsod,
    target=data.reference