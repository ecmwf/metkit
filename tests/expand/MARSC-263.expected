RETRIEVE,
    CLASS      = OD,
    TYPE       = OF,
    STREAM     = OCEA,
    EXPVER     = 0001,
    REPRES     = SH,
    LEVTYPE    = DP,
    LEVELIST   = 0/5,
    PARAM      = 129/130/145,
    DATE       = 20120101,
    TIME       = 0000,
    STEP       = 24/48/72/96,
    NUMBER     = 0/1/2/3/4,
    DOMAIN     = G,
    SYSTEM     = 3,
    METHOD     = 1,
    PRODUCT    = INST,
    SECTION    = H,
    TARGET     = data.reference,
    DATABASE   = marsod,
    EXPECT     = 60